src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+import mate.academy.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order add(Order order) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(order);
+            transaction.commit();
+            return order;
+        } catch (RuntimeException e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't add order to DB, order: "
+                    + order, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+
+    }
+
+    @Override
+    public List<Order> getByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Order> query = session
+                    .createQuery("from Order o "
+                            + "left join fetch o.tickets t "
+                            + "left join fetch t.movieSession ms "
+                            + "left join fetch ms.movie m "
+                            + "left join fetch ms.cinemaHall "
+                            + "left join fetch o.user  "
+                            + "where o.user = :user", Order.class);
```

Comment:
It is better to write all commands in upper case FROM, LEFT JOIN FETCH, WHERE...